### PR TITLE
Added breadcrumb-file creation condition as attribute

### PR DESCRIPTION
**Pull request by paulpalamarchuk**

### CODE REVIEW

**Comment by chewi** [comment]
I wasn't familiar with redis-sentinel so I was quite surprised that my configuration file was not updated by Chef. Please make this optional.

**Comment by kam1kaze** [comment]
@chewi @shortdudey123 could you take a look to this PR. We have to use own fork due to such limitation.

**Comment by shortdudey123** [comment]
@paulpalamarchuk can you rebase on master?
Otherwise i can carry this commit into another PR

**Comment by paulpalamarchuk** [comment]
@shortdudey123 rebased

File: README.md [hunk 1]
```diff
@@ -366,7 +381,9 @@ Available options and their defaults
 'cluster-enabled'            => 'no',
 'cluster-config-file'        => nil, # Defaults to redis instance name inside of template if cluster is enabled.
 'cluster-node-timeout'       => 5000,
-'includes'                   => nil
+'includes'                   => nil,
+'breadcrumb'              => true # Defaults to create breadcrumb lock-file.
```
Comment (shortdudey123) [inline README.md]: Can you fix the spacing here?

**Comment by shortdudey123** [comment]
small nit pick in spacing then will merge

**Comment by paulpalamarchuk** [comment]
@shortdudey123 done

**Comment by shortdudey123** [comment]
thanks!

**Comment by kam1kaze** [comment]
Yeah!

thank you all! 👍

**Pull request merged**